hummingbot/client/command/silly_commands.py

```diff
@@ -41,11 +41,8 @@ def be_silly(self,  # type: HummingbotApplication
 
```

Comment:
One of the main aspects of silly commands is a kind of a personal private space (your own command and your own code) within the public open source Hummingbot. So the approach to modify this, when it's not your command, is to apply minimum changes (the kind that without it, the command would fail). There is no need to optimise, nor correct timing.